### PR TITLE
[full-ci] [tests-only] Fixed failure in lock test

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
The file is opened with `Collabora` and `then` step is checked with `OnlyOffice` so the selector of `OnlyOffice` is not found in `collabora` editor. 
The failure arose after this PR https://github.com/owncloud/web/pull/10873 .

## Related Issue

- Fixes https://github.com/owncloud/web/issues/10880

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

**Review by saw-jan — APPROVED** [review]

**Comment by saw-jan** [comment]
>The failure arose after this PR https://github.com/owncloud/web/pull/10873 .

Ah, PR doesn't have full-ci in the title so we didn't catch it.

**Comment by grgprarup** [comment]
The failure is due to incomplete implementation of step `And "Alice" should be able to manage share with user "Brian"`, the sidebar panel of share is not open.
```sh
Then(
  /^"([^"]*)" (should|should not) be able to manage share with user "([^"]*)"$/,
  async function (
    this: World,
    stepUser: any,
    actionType: string,
    recipient: string
  ): Promise<void> {
    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
    const shareObject = new objects.applicationFiles.Share({ page })
    const changeRole = shareObject.changeRoleLocator(
      this.usersEnvironment.getUser({ key: recipient })
    )
    const changeShare = shareObject.changeShareLocator(
      this.usersEnvironment.getUser({ key: recipient })
    )

    if (actionType === 'should') {
      await expect(changeRole).not.toBeDisabled()
      await expect(changeShare).not.toBeDisabled()
    } else {
      await expect(changeRole).toBeDisabled()
      await expect(changeShare).toBeDisabled()
    }
  }
)
```

File: tests/e2e/cucumber/features/app-provider/lock.feature [hunk 1]
```diff
@@ -34,7 +34,7 @@ Feature: lock
 
     # checking that user cannot delete or change share of the locked file
     # https://github.com/owncloud/web/issues/10507
-    And "Alice" should not be able to manage share with user "Brian"
+    And "Alice" should not be able to manage share for file "test.odt" with user "Brian"
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/app-provider/lock.feature:37]:
```suggestion
    And "Alice" should not be able to manage share of a file "test.odt" for user "Brian"
```

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**